Solutions/Corelight/Package/mainTemplate.json

```diff
@@ -28,2891 +28,3952 @@
         "description": "Workspace name for Log Analytics where Microsoft Sentinel is setup"
       }
     },
+    "formattedTimeNow": {
+      "type": "string",
+      "defaultValue": "[utcNow('g')]",
+      "metadata": {
+        "description": "Appended to workbook displayNames to make them unique"
+      }
+    },
+    "workbook1-id": {
+      "type": "string",
+      "defaultValue": "[newGuid()]",
+      "minLength": 1,
+      "metadata": {
+        "description": "Unique id for the workbook"
+      }
+    },
     "workbook1-name": {
       "type": "string",
       "defaultValue": "Corelight",
       "minLength": 1,
       "metadata": {
         "description": "Name for the workbook"
       }
-    }
-  },
-  "variables": {
```

Comment:
Instances set under this parameter ( variables )  are template variabels which are crossing the limit of 512 
<img width="958" alt="MicrosoftTeams-image (20)" src="https://github.com/Azure/Azure-Sentinel/assets/117061676/45b24e32-af64-4451-97a0-114776fa1902">